wg-arch-arm/charter.md

```diff
@@ -0,0 +1,31 @@
+# WG ARM architecture Charter
+
+## Scope
+
+The ARM architecture working group takes care of all things related to supporting ARM architecture on KubeVirt clusters.
+This includes all build processes, also support of failing builds or test lanes.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- list of what qualifies a piece of code, binary or service
```

Comment:
@zhlhahaha hey, here's the charter for the wg arch arm, please chime in here with the things you want to have added, I'll happily integrate them

Reply:
I'm unsure about what should be included in the "In Scope" section. Perhaps we can follow the same outlines used for x86_64.

Regarding the "Out of Scope" section, I believe we currently only support KVM-based emulation on Arm64 servers. Therefore, binary-based emulation and services on Arm32 are not within the scope.